### DISCUSSION TITLE
Preview Data

### DISCUSSION THREAD

**Discussion by databayes** [TabularEditor/TabularEditor3#1307] [Ideas]
### Is your feature request related to a problem? Please describe

I am previewing table data frequently, is there an option to double click in the top left square to expand all the columns?

### Describe the solution you'd like

To be able to expand all columns at once
![delete](https://github.com/TabularEditor/TabularEditor3/assets/87519321/5df6e539-d259-4603-a98a-becfca1a3a32)

### Goals

1. Visibility of data
2.

**Comment by databayes** [comment]
Found it - Best Fit

**Comment by otykier** [comment]
@mlonsk can you add the following suggestions to the backlog:

- Perform a "best fit" when double clicking the empty square to the left of column headers
- Change the behavior of the "Collapse all" button in the TOM Explorer, so that it alternates between collapsing all notes and restoring the expand state

(Not sure if we need to convert this discussion to GitHub issues?)
